geode-protobuf/src/distributedTest/java/org/apache/geode/internal/protocol/protobuf/v1/operations/GetAndPutJsonDocumentsDUnitTest.java

```diff
@@ -121,20 +131,20 @@ public void testThatGetReturnsJSONDocumentForPdxInstance() throws Exception {
       String actualValue = response.getResult().getJsonObjectResult();
       assertEquals(jsonDocument, actualValue);
     } finally {
-      getCache().setReadSerializedForCurrentThread(false);
+      cache.setReadSerializedForCurrentThread(false);
```

Comment:
I recommend moving this line to tearDown() method and just delete the try-finally blocks from all of these tests. Helps reduce the noise.

Reply:
I think I disagree on this point. I prefer to have tests clean up for themselves anything that's not part of the setup/teardown scope. I don't really want to have _all_ the tests run with read-serialized set to true, and I would prefer not to decouple the scope of setup and cleanup for the thread-local read-serialized.